history-service/src/storage/attempt_store.ts

```diff
@@ -145,6 +146,58 @@ class AttemptStore implements IAttemptStore {
       totalCount,
     };
   }
+
+  async removeHistoryOwner(userId: number): Promise<void> {
```

Comment:
Suggest renaming the function to make the purpose clearer.
```suggestion
  async removeAllOfHistoryOwner(userId: number): Promise<void> {
```